apps/native-component-list/src/navigation/ExpoComponentsStackNavigator.tsx

```diff
@@ -409,6 +409,12 @@ export const Screens: ScreenConfig[] = [
     },
     name: 'Video (expo-video)',
   },
+  {
+    getComponent() {
+      return optionalRequire(() => require('../screens/VideoThumbnailsScreen'));
+    },
+    name: 'Video Thumbnails',
+  },
```

Comment:
`expo-video-thumbnails` is not a component, can you please move it to `ExpoApisStackNavigator` instead?

Reply:
Done